open-company/open-company-web
# Chart tweaks

card: https://trello.com/c/PMy2Iplk

To test:
- check all the finances metrics for all the demo companies
- check all the growth metrics for all the demo companies
